### ISSUE TITLE
vext.pyqt4 doesn't work

### ISSUE THREAD

**Issue by rutsky** [stuaxo/vext#17]
To reproduce on Ubuntu 14.04 with python 3.4.2 with  system-wide installed PyQt4 (from python3-pyqt4 package version 4.10.4):

```
$ python3
Python 3.4.3 (default, Mar 26 2015, 22:03:40) 
[GCC 4.9.2] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import PyQt4
>>> 
$ python3 -m venv env
$ source env/bin/activate
(env) $ pip install vext.pyqt4
Downloading/unpacking vext.pyqt4
  Downloading vext.pyqt4-0.2.1.tar.gz
  Running setup.py (path:/home/bob/tmp/18/env/build/vext.pyqt4/setup.py) egg_info for package vext.pyqt4

    warning: no files found matching '*.rst'
Downloading/unpacking vext (from vext.pyqt4)
  Downloading vext-0.3.11.tar.gz
...
Successfully installed vext.pyqt4 vext pyyaml
Cleaning up...
(env) $ python 
Python 3.4.3 (default, Mar 26 2015, 22:03:40) 
[GCC 4.9.2] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import PyQt4
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/bob/tmp/18/env/lib/python3.4/site-packages/vext-0.3.11-py3.4.egg/vext/__init__.py", line 139, in load_module
    raise ImportError("No module named %s" % basename)
ImportError: No module named PyQt
>>>
(env) $
```

**Comment by stuaxo** [comment]
python3 is not supported yet, due to differences in the import system.

https://github.com/stuaxo/vext/issues/1

There is another bug in that the install shouldn't have got that far.

**Comment by stuaxo** [comment]
Hrm, just noticed I have supports python 3.4 in the setup.py :/

**Comment by stuaxo** [comment]
Closed as dupe of #18 and #1
